configs/next/packages/glibc/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="GNU C Library"
 ATSRC_PACKAGE_VER=2.31.9000
-ATSRC_PACKAGE_REV=08538f360f14
+ATSRC_PACKAGE_REV=7ebd114211dc
```

Comment:
Did you intend to include this change in this pull request?

Reply:
@ThinkOpenly Yes. This was the change that actually brought up the whole problem. See #1570. So I added it here.

Reply:
I guess it seems a little out of place in this pull request, but not a big deal.